telegram/_message.py

```diff
@@ -576,6 +577,10 @@ class Message(MaybeInaccessibleMessage):
         paid_media (:obj:`telegram.PaidMediaInfo`, optional): Message contains paid media;
             information about the paid media.
 
+            .. versionadded:: NEXT.VERSION
+        refunded_payment (:obj:`telegram.RefundedPayment`, optional): Message is a service message
+            about a refunded payment, information about the payment.
+
```

Comment:
```suggestion
            about a refunded payment, information about the payment.

            .. seealso::
                `Telegram Docs on payments <https://core.telegram.org/bots/api#payments>`_
```

Reply:
While we could definitely add this, it would also need to be added to successful_payment and paid_media.

All of them getting this big block might take up unnecessary amount of screen time, I would do an inline link then instead

Reply:
hmm I was just following what hinrich started with https://github.com/python-telegram-bot/python-telegram-bot/pull/4303/commits/027263a223a13167d33d9f582a5c701ba860041b, inline link also works with me

Reply:
![image](https://github.com/python-telegram-bot/python-telegram-bot/assets/25934244/2ae09019-926e-46ec-b50a-52dbffe9c063)

I think that is really a lot of wasted screen space